0000-template.md

```diff
@@ -0,0 +1,73 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Amethyst Issue: (leave this empty)
+
+# Summary
```

Comment:
Another request, can we use HTML elements (collapsible summaries / anchor links) to make navigation easier?

Reply:
Done